ee/tabby-ui/lib/utils.ts

```diff
@@ -78,3 +78,13 @@ export const delay = (ms: number) => {
     setTimeout(() => resolve(null), ms)
   })
 }
+
+export const convertToDisplayName = (lan: string): string => {
```

Comment:
take graphql generated type as input.

Reply:
Update other places to use correct `Language` to replace `string` or `keyof Language`